docs/examples/use_cases/webdataset-externalsource.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.                                    *len(extensions))
</pre><p>Please adjust the expression so it doesnt mix <code>*</code>  as multiplication and unpacking the tuple.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>